.github/workflows/integration.yml

```diff
@@ -0,0 +1,138 @@
+name: Integration
+run-name: ${{ inputs.UUID }}
+
+on:
+  push: # TODO: remove this
+    branches:
+      - tests/integration-tests 
+  workflow_dispatch:
+    inputs:
+      scheduler_res_path:
+        type: string
+        description: path to test resources to use for this run
+        required: true
+        default: tests/fixtures/01
+      UUID:
```

Comment:
I'd use lowercase `id` and document that it should be a uuid instead.

Reply:
renaming this field to run_name, it makes a lot of things less ambiguous